azext_iot/central/_help.py

```diff
@@ -44,6 +57,158 @@ def load_central_help():
     _load_central_monitors_help()
     _load_central_command_help()
     _load_central_compute_device_key()
+    _load_central_export_help()
+
+
+def _load_central_export_help():
+    helps[
+        "iot central export"
+    ] = """
+        type: group
+        short-summary: Manage and configure IoT Central exports.
+    """
+
+    helps[
+        "iot central export list"
+    ] = """
+        type: command
+        short-summary: Get the list of exports for an IoT Central application.
+        examples:
+        - name: List all exports in an application
+          text: >
+            az iot central export list
+            --app-id {appid}
+    """
+
+    helps[
+        "iot central export show"
+    ] = """
+        type: command
+        short-summary: Get an export details
+        examples:
+        - name: Get an export details
+          text: >
+            az iot central export show
+            --app-id {appid}
+            --export-id {exportid}
+    """
+
+    helps[
+        "iot central export create"
+    ] = """
+        type: command
+        short-summary: Create an export for an IoT Central application.
+        examples:
+        - name: Create an export
+          text: >
+            az iot central export create
+            --app-id {appid}
+            --export-id {exportid}
+            --content {content}
+    """
+
+    helps[
+        "iot central export update"
+    ] = """
+        type: command
+        short-summary: Update an export for an IoT Central application.
+        examples:
+        - name: Update an export
+          text: >
+            az iot central export update
+            --app-id {appid}
+            --export-id {exportid}
+            --content {content}
+    """
+
+    helps[
+        "iot central export delete"
+    ] = """
+        type: command
+        short-summary: Delete an export for an IoT Central application.
+        examples:
+        - name: Delete an export
+          text: >
+            az iot central export delete
+            --app-id {appid}
+            --export-id {exportid}
+    """
+    _load_central_destination_help()
+
+
+def _load_central_destination_help():
+    helps[
+        "iot central export destination"
+    ] = """
+        type: group
+        short-summary: Manage and configure IoT Central destinations.
+    """
+
+    helps[
+        "iot central export destination list"
+    ] = """
+        type: command
+        short-summary: Get the list of destinations for an IoT Central application.
```

Comment:
Think it would be a bit more clear to prepend "export" to destination in the help descriptions i.e. `Get the list of export destinations for an IoT Central application.`

Reply:
Same applies for other destination commands `Get a destination details` -> `Get an export destination details` etc

Reply:
Fixed as suggested